homeassistant/components/device_tracker/__init__.py

```diff
@@ -125,9 +126,11 @@ def async_setup(hass: HomeAssistantType, config: ConfigType):
     conf = conf[0] if conf else {}
     consider_home = conf.get(CONF_CONSIDER_HOME, DEFAULT_CONSIDER_HOME)
     track_new = conf.get(CONF_TRACK_NEW, DEFAULT_TRACK_NEW)
+    hide_if_away = conf.get(CONF_AWAY_HIDE, DEFAULT_AWAY_HIDE)
```

Comment:
This is a very confusing name. It's not `hide_if_away`, it's the default value for hide_if_away for newly discovered devices.

We should probably create a new object to hold values for newly discovered devices. It can hold track, hide_if_away. 

Also, adding it as a configuration option named `hide_if_away` is confusing too. 

Actually, instead of this just add a new option with `new_device_defaults` and have that be applied to each new discovered device.

Reply:
I agree that it's confusing name. May be it will be better to add optional object 'initial_value' that will hold defaults for new devices.

Reply:
@balloob Moving `track_new_devices` to `new_device_defaults` object will be braking change. What do you think of leaving it as is? Later I will create new PR with optional `new_device_defaults` that will hold new discovered device options

Reply:
You can leave track_new for now and support both track_new and new_device_defaults. We should not add a new `hide_if_away` option.